### DISCUSSION TITLE
Misc Error Messages

### DISCUSSION THREAD

**Discussion by tootbrute** [photoprism/photoprism#1116] [Q&A] [answered]
Can anybody shed some light on these error messages? I haven't seen these types of error messages before.

time="2021-03-04T10:20:35Z" level=warning msg="Error 1062: Duplicate entry '2021/02/20210201_105442.jpg-/' for key 'idx_files_name_root'"

time="2021-03-04T10:18:55Z" level=warning msg="metadata: scan-data is unbounded; EOI not encountered before EOF in 20140510_121615_86BCE5FE.jpg (parse jpeg)"

time="2021-03-04T10:18:55Z" level=error msg="media: failed creating thumbnail for 20140510_121615_86BCE5FE.jpg (invalid JPEG format: short Huffman data)"

**Comment by lastzero** [comment, marked as answer]
> time="2021-03-04T10:20:35Z" level=warning msg="Error 1062: Duplicate entry '2021/02/20210201_105442.jpg-/' for key 'idx_files_name_root'"

Seems the same file was indexed twice... maybe really a "duplicate" like `2021/02/20210201_105442.jpg` and `2021/02/20210201_105442.JPG`? Wouldn't know how else this might happen...

> time="2021-03-04T10:18:55Z" level=warning msg="metadata: scan-data is unbounded; EOI not encountered before EOF in 20140510_121615_86BCE5FE.jpg (parse jpeg)"

Broken EXIF data. Happens, nothing we can do about.

> time="2021-03-04T10:18:55Z" level=error msg="media: failed creating thumbnail for 20140510_121615_86BCE5FE.jpg (invalid JPEG format: short Huffman data)"

Broken JPEG data. Happens, nothing we can do about.
